content/source/docs/cloud/workspaces/variables.html.md

```diff
@@ -66,6 +64,15 @@ module "ec2_instances" {
 
 If a required input variable is missing, Terraform plans in the workspace will fail and print an explanation in the log.
 
+#### Loading Variables from the Command Line
+
+When using Terraform 1.1 or later, you can set input variable values for a particular plan/apply run on the command line using either of the following mechanisms:
+
+* Specifying `-var` and `-var-file` arguments
+* Local environment variables named `TF_VAR_` followed by the name of a declared variable
+
+Variable values set in either of these ways will take precedence over Workspace-Specific values. See the [Precedence](#precedence) section for more details. For more information about specifying CLI input variable values, see the Terraform documentation for [Variables on the Command Line](/docs/language/values/variables.html#variables-on-the-command-line) or for the [Input Variables on the Command Line](/docs/cli/commands/plan.html#input-variables-on-the-command-line) for the plan command.
```

Comment:
So, I think that you can only have TFC pick up a `terraform.tfvars` file if you do this command line flag, right? It doesn't work the same way as `auto.tfvars`, right? Like, if I try to to provide TFC with a `terraform.tfvars` file the same way I would with `auto.tfvars`, does it recognize it? We should probably be explicit about if that does or does not work and why.

```suggestion
Variable values set in either of these ways will take precedence over Workspace-Specific values. See the [Precedence](#precedence) section for more details. For more information about specifying CLI input variable values, see the Terraform documentation for [Variables on the Command Line](/docs/language/values/variables.html#variables-on-the-command-line) or for the [Input Variables on the Command Line](/docs/cli/commands/plan.html#input-variables-on-the-command-line) for the plan command.
```

Reply:
TFC will actually overwrite your terraform.tfvars file with one of it's own, so those values are completely ignored by TFC. For some reason, I thought we documented that all over the place but I don't see it anywhere in the doc! I'll add it.